src/foam/u2/wizard/axiom/AnalyticWizardAction.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+  
+ foam.CLASS({
+    package: 'foam.u2.wizard.axiom',
+    name: 'AnalyticWizardAction',
+    extends: 'foam.u2.wizard.axiom.WizardAction',
+  
+    documentation: 'WizardAction with analytic events',
+  
+    imports: [
+      'analyticsAgent?'
```

Comment:
Right way to log events now is to implement Analyticable interface and use the report function